settings/docker.yml

```diff
@@ -33,6 +33,9 @@ sidekiq_redis_url: redis://redis:6379/2
 efs_submissions_mount_point: "/tmp/exercism/efs/submissions"
 efs_repositories_mount_point: "/tmp/exercism/efs/repos"
 
+# GitHub Config
+github_organization: "exercism-test"
```

Comment:
What would the correct name be for the github organization? I just used a dummy value here.

Reply:
Should we make one?